pandas/core/nanops.py

```diff
@@ -637,7 +619,7 @@ def _maybe_null_out(result, axis, mask):
             else:
                 result = result.astype('f8')
             result[null_mask] = np.nan
-    else:
+    elif result is not tslib.NaT:
```

Comment:
== tslib.iNaT

but puzzled why a NaT is there
as I don't think it's wrapped yet